tests/io_test.py

```diff
@@ -261,41 +260,45 @@ def testSaveStringsDataset(self):
                             format(IOTest.io_test_dir), dsetName='strings/values'))
         self.assertIsNotNone(ak.read_hdf(filenames='{}/strings-test_LOCALE0'.\
                             format(IOTest.io_test_dir), dsetName='strings/segments'))
-
-    @unittest.skip       
+     
     def testSaveLongStringsDataset(self):
         # Create, save, and load Strings dataset
-        strings_array = ak.array(['testing a longer string{} to be written, loaded and appended'.\
-                                  format(num) for num in list(range(1,11))])
-        strings_array.save('{}/strings-test'.format(IOTest.io_test_dir), dataset='strings')
-        r_strings_array = ak.load('{}/strings-test'.format(IOTest.io_test_dir), 
-                                  dataset='strings')
-        self.assertTrue((strings_array == r_strings_array).all())       
+        strings = ak.array(['testing a longer string{} to be written, loaded and appended'.\
+                                  format(num) for num in list(range(0,26))])
+        strings.save('{}/strings-test'.format(IOTest.io_test_dir), dataset='strings')
+        n_strings = strings.to_ndarray().sort()
+        r_strings = ak.load('{}/strings-test'.format(IOTest.io_test_dir), 
+                                  dataset='strings').to_ndarray().sort()
+        self.assertTrue((n_strings == r_strings))       
```

Comment:
Should use `np.sort()` and `.all()` (see above).

Reply:
Fixed